src/client/components/Registration.jsx

```diff
@@ -1,4 +1,44 @@
+import { useState } from "react";
+
+
+const emptyUser = {
+    email: "",
+    username: "",
+    password: ""
+}
+
+const registerURL = "http://localhost:4000";
+
 const Registration = () => {
+    const [userDetails, setUserDetails] = useState(emptyUser);
```

Comment:
Been a while since we've used state! Nice memory :)

Reply:
Thanks Lenni. Elephants never forget.